analyses/summary/helper_test.go

```diff
@@ -30,7 +30,7 @@ func data() driver.GetResponseType {
 
 func TestProcessSummary(t *testing.T) {
 	got := ProcessSummary(data(), false)
-	if diff := cmp.Diff(false, got); diff != "" {
+	if diff := cmp.Diff(true, got); diff != "" {
```

Comment:
I've missed this, but why do you need to use cmp.Diff just for boolean checks? It is okay to use for struct comparision.

Reply:
Fixed. Thanks